### ISSUE TITLE
Export of PLOT3D File in FDS 6.9.0 with incorrect variable names

### ISSUE THREAD

**Issue by rupole1185** [firemodels/fds#12668]
**Describe the bug**
I am exporting the plot3D geometry file using 
_&DUMP NFRAMES=600., WRITE_XYZ=.TRUE., PLOT3D_QUANTITY(1:5)='TEMPERATURE','U-VELOCITY','V-VELOCITY','W-VELOCITY','VISIBILITY',_

Once I open the files in [ParaView](https://www.paraview.org/download/) they come out with variable names not matching the PLOT3D_QUANTITY I selected

**To Reproduce**
Run the case: [STARICASE.txt](https://github.com/firemodels/fds/files/14726191/STARICASE.txt)
When the simulation finishes open the XYZ file using paraview 5.10 or later
Import the .q file that belongs to the mesh loaded
The variable names do not match the exported ones

**Expected behavior**
I would expect to have the following variable in paraview:
'TEMPERATURE'
'VELOCITY'
'VISIBILITY'

Instead I can see these:

![image](https://github.com/firemodels/fds/assets/7260681/6c1d0361-877c-479b-9f2a-ebeb1772cee7)

where: 

Density -> Temperature
Momentum -> Velocity
StagnationEnergy -> Smoke Visibility

**Comment by mcgratta** [comment]
I don't think we have control over this. The names that Paraview lists are the names of the quantities that Plot3D was originally intended to carry. Plot3D is a format developed by NASA to hold generic CFD data.

**Comment by mcgratta** [comment]
By the way, in past versions, did you see the "FDS names" of the quantities in Paraview?

**Comment by rupole1185** [comment]
Hi Kevin, I have not tried versions before 5.10 of Paraview. I read some documentation about plot3D and it seems that the .q files have by standard they have Density, Momentum and Energy (https://daac.hpc.mil/gettingStarted/Plot3D.html). Somehow we are generating the "Pressure" variable which is non standard though, so it should be possible to add more variables. I try to investigate more on the documentation even if it is a bit old.

**Comment by mcgratta** [comment]
Plot3d was developed for aerospace CFD codes, for which density, three components of momentum, and internal energy are the conserved scalars. We do not use these defaults.

**Comment by johodges** [comment]
There are a few other options you could take if you want to visualize your data in paraview.

1. Compile the unofficial vtk-out fork on which I have been working (https://github.com/johodges/fds/tree/vtk-out). If you set "DT_VTK" on the DUMP line this fork will write out parallel versions of VTK files. If you want the full 3-D domain like you would get with a plot3d file, you will need to add 3-D slices of the domain (see attached input file). There will also be a python (.py) paraview state file written out which you can use to auto configure the data. Note the auto-generated state file will work with paraview 5.12 but will get some errors with older versions due to the devs renaming some of the attributes.
2. Convert your binary data from a previous FDS run to VTK files using pyfdstools (https://github.com/johodges/pyfdstools/blob/master/pyfdstools/example_paraview_export.py).
3. Use the native fds reader for paraview which came out in the latest release. It's a bit buggy with multi-meshes and does not allow you to visualize the smoke 3d files, but it may work for your needs as well.

I tried approach 1 with your input file and everything seems to be working correctly. Let me know if you choose to try it out and run into issues. I will address them as they come up.

![image](https://github.com/firemodels/fds/assets/17127980/c1da0a94-c7b2-45e4-80c0-31ec8bdd31e4)
[staircase.txt](https://github.com/firemodels/fds/files/14733270/staircase.txt)

**Comment by rupole1185** [comment]
Having a VTK export would be awesome as I tried the native reader they implemented in paraview and it not ideal (it just gives info on the slices the user enters and not all over the domain as the plot3d currently does).
@mcgratta wouldn't be worth a while merging this feature?

**Comment by mcgratta** [comment]
We are currently working on various VTK exports. Plot3d is a good first case. The downside of slice and boundary files is that each time step requires a separate VTK file.

**Comment by johodges** [comment]
I did get a clean firebot run last night on the vrk-out branch which is up to date with master. However, I was planning to wait to get it merged until after you guys have the new cluster up and running. I also want to talk with the kitware team a bit about the configuration of the state file to streamline user experience a bit.

**Comment by rupole1185** [comment]
@mcgratta it may use a bit of the hard drive, but in paraview once you have the VTK of the whole 3D domain, you do not need slices as you can extract them directly in paraview. Or alternatively you can use a multi-block dataset.

**Comment by johodges** [comment]
Closing out this issue as there is not an error in the PLOT3D output. VTK outputs are available in the FireX fork of FDS currently. The output format is in a bit of flux and is being discussed in https://github.com/firemodels/fds/issues/14755.
